lib/pka_dev.c

```diff
@@ -843,58 +843,6 @@ static int pka_dev_delete_shim(pka_dev_shim_t *shim)
     return ret;
 }
 
-static int pka_dev_config_aic_interrupts(pka_dev_res_t *aic_csr_ptr)

```

Comment:
There are resources and pointers to resources that are defined but no longer used. I think we should clean them up as well.

Reply:
Agreed. Some of the firmware image related variables start to generate build error with a clean build.